tools/ARIAtools/extractProduct.py

```diff
@@ -341,8 +394,8 @@ def merged_productbbox(metadata_dict, product_dict, workdir='./', bbox_file=None
 
     return metadata_dict, product_dict, bbox_file, prods_TOTbbox, arrshape, proj
 
-
-def export_products(full_product_dict, bbox_file, prods_TOTbbox, layers, rankedResampling=False, dem=None, lat=None, lon=None, mask=None, outDir='./',outputFormat='VRT', stitchMethodType='overlap', verbose=None, num_threads='2', multilooking=None):
+def export_products(full_product_dict, bbox_file, prods_TOTbbox, layers, rankedResampling=False, dem=None, lat=None, lon=None, mask=None, \
```

Comment:
Codacy found an issue: [export_products is too complex (17) (MC0001)](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5789558)